src/geo/map-view.js

```diff
@@ -152,10 +212,23 @@ var MapView = View.extend({
       return;
     }
 
-    var layerView = this._createLayerView(this._cartoDBLayerGroup);
-    this._cartoDBLayerGroupView = layerView;
-    this._layerViews[layerModel.cid] = layerView;
-    return layerView;
+    // Create the view that groups CartoDB layers
+    this._cartoDBLayerGroupView = this._createLayerView(this._cartoDBLayerGroup);
+    this._layerViews[layerModel.cid] = this._cartoDBLayerGroupView;
+
+    // Render the infowindow and tooltip "overlays"
+    this._infowindowView.render();
+    this.$el.append(this._infowindowView.el);
+    this._tooltipView.render();
+    this.$el.append(this._tooltipView.el);
+
+    // Start managers that should be bound to the CartoDB layer group view
+    this._infowindowManager.start(this._cartoDBLayerGroupView);
```

Comment:
As discussed with @rochoa, this would be a nice improvement, but it's not trivial so we'll tackle it in the future.